### ISSUE TITLE
README.md should include `npm run prepare-java` (etc.)

### ISSUE THREAD

**Issue by mrienstra** [microsoft/playwright.dev#430]
While trying to contribute, I followed instructions in the [README.md](https://github.com/microsoft/playwright.dev/blob/main/README.md):

```sh
npm install
SRC_DIR=~/code/playwright npm run roll
npm run start
```

... But this resulted in `dotnet`-only docs being served.

In my case, it seemed I needed to run `npm run prepare-nodejs` in order to see `nodejs` docs.

Seems like this would be worth mentioning, to help newbies get up and running quickly.

PS: Also tempted to change `~/code/playwright` to `~/path/to/playwright` or something... 🤔

### Self-service
☑ I'd be willing to fix this bug myself.

**Comment by mxschmitt** [comment]
Yeah setting up the docs locally is not that straightforward. Usually you: npm run roll -> npm run prepare-nodejs -> npm run start e.g.

**Comment by mrienstra** [comment]
Along the same lines, what about setting the default (shown when a `prepare-` command has not been run) to `nodejs` rather than `dotnet`, as per the website itself (which defaults to showing "Node.js" docs)?

Not immediately obvious to me how to go about changing that.

**Comment by mxschmitt** [comment]
Done in https://github.com/microsoft/playwright.dev/commit/26dd38bbbc6e3bc463187c731ee7fe01b9d9828a
